utils/api/request.ts

```diff
@@ -78,5 +78,11 @@ export const httpRequest = async <T>(
     handleRequestError(res);
   }
 
-  return res.json() as T;
+  const text = await res.text();
```

Comment:
Not sure why this change is needed. Any explanation for that?

Reply:
Thanks for noticing. This change was implemented to avoid an error in the context of form submit but is no longer relevant (the error does not occur anymore). I have reverted the changes.

Reply:
Sorry. Just had a quick test again and the like function was broken. A like action will lead to an error that i do not fully understand but that the mentioned change suppresses.

With the change the request/action returns 

0:["$@1",["development",null]]
1:"$undefined"

and without the change there will be a 500 internal server error.

Reply:
Fair enough. I will approve it for now. Can you then create a ticket with this description so we can tackle this issue?